src/elements/kw-share-feed/kw-share-feed.html

```diff
@@ -58,14 +65,29 @@
             }
         </style>
         <div class="shell" id="shell">
-            <kw-share-card share="[[shellShare]]"></kw-share-card>
-            <kw-share-card share="[[shellShare]]"></kw-share-card>
+            <!--<kw-share-card share="[[shellShare]]"></kw-share-card>
+            <kw-share-card share="[[shellShare]]"></kw-share-card>-->
```

Comment:
Don't leave commented code. Just remove it, we can always get it back with git if we need